src/providers/tfc/organizationPicker.ts

```diff
@@ -16,7 +16,7 @@ export class CreateOrganizationItem implements vscode.QuickPickItem {
     return 'Open the browser to create a new organization';
   }
   async open() {
-    await vscode.env.openExternal(vscode.Uri.parse(`https://${TerraformCloudHost}/app/organizations`));
+    await vscode.env.openExternal(vscode.Uri.parse(`https://${TerraformCloudWebUrl}/organizations`));
```

Comment:
```suggestion
    await vscode.env.openExternal(vscode.Uri.parse(`${TerraformCloudWebUrl}/organizations`));
```
Otherwise the `https://` part would be duplicated here, wouldn't it?